ci/build.ps1

```diff
@@ -13,6 +13,7 @@ $buildDir = [System.IO.Path]::GetFullPath("$(pwd)")
 $depsCmakeVars = @{
   CMAKE_BUILD_TYPE = $cmakeBuildType;
 }
+$cmakeExtraFlags = $env:CMAKE_EXTRA_FLAGS
```

Comment:
don't need this variable

Reply:
As a side note, I like the "EXTRA" because it expresses that those area additional flags and not the full set, not overriden the others. No matter though :)

I'll remove this variable and instead use `$env:CMAKE_FLAGS` in the make invocation. Should I go through the docs to change `CMAKE_EXTRA_FLAGS` to `CMAKE_FLAGS`? The wiki? Does it need to be changed somewhere else?

Reply:
> Should I go through the docs to change `CMAKE_EXTRA_FLAGS` to `CMAKE_FLAGS`? The wiki?

It's not fully transitioned in the Makefile.